ext/redhat/pxp-agent.init

```diff
@@ -55,6 +55,15 @@ stop() {
             RETVAL=$?
         fi
     fi
+
+    pids=$(pgrep -u `whoami` -x $prog)
```

Comment:
When the file is absent, `killproc` might fail and we don't get to this point. Although I'm not sure why it'd do that rather than capturing the status and returning it at line 69.

Reply:
Does this latest version work with manual testing?